### DISCUSSION TITLE
LBM convergence criteria

### DISCUSSION THREAD

**Discussion by nahidrahmati** [UvaCsl/HemoCell#118] [General discussion]
Hi, 
I am using Hemocell for simulating deformable cells in blood plasma. However, I have a question regarding LBM solver which is related to palabos. I want to know the convergence criteria for LBM solver? and if it is possible can we change it?

**Comment by gzavo** [comment]
Hi @nahidrahmati ,

Your question seems underdefined (at least for me it is not clear what are you looking for).
HemoCell is a combination of 3 numerical methods: LBM, IBM, DEM. They all have different stability and accuracy profiles that contribute to HemoCell, depending on the parameters (flow velocity, hematocrit, geometry, etc.).
